x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/add_exception_flyout/index.tsx

```diff
@@ -106,6 +107,7 @@ export const AddExceptionFlyout = memo(function AddExceptionFlyout({
   showAlertCloseOptions,
   isAlertDataLoading,
   alertStatus,
+  sharedListToAddTo,
```

Comment:
sending the list that the Add Exception Flyout is already opened from, to disable the `Add-To-Rule/ Shared List` part in the Shared Lists page as well as in the Details page